packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -106,24 +105,10 @@ export function FlatTermSelector( { slug } ) {
 
 			return {
 				hasCreateAction: _taxonomy
-					? get(
-							post,
-							[
-								'_links',
-								'wp:action-create-' + _taxonomy.rest_base,
-							],
-							false
-					  )
+					? post._links?.[ 'wp:action-create-' + _taxonomy.rest_base ]
 					: false,
 				hasAssignAction: _taxonomy
-					? get(
-							post,
-							[
-								'_links',
-								'wp:action-assign-' + _taxonomy.rest_base,
-							],
-							false
-					  )
+					? post._links?.[ 'wp:action-assign-' + _taxonomy.rest_base ]
```

Comment:
`getCurrentPost()` will always be an object:

https://github.com/WordPress/gutenberg/blob/4944315cac156c78313121657dff9d58ce8f4d3f/packages/editor/src/store/selectors.js#L170